cmd/image/sync/mock/sync_client_generated.go

```diff
@@ -33,6 +33,7 @@ import (
 type MockSyncClient struct {
 	ctrl     *gomock.Controller
 	recorder *MockSyncClientMockRecorder
+	isgomock struct{}
```

Comment:
why is this getting added?

Reply:
It gets created while using mockgen , to help tell the difference between the real object and the fake (mock) version used for testing.
Can be removed as well.

Reply:
Addl ref: 
https://github.com/uber-go/mock/commit/33ccc990781d28f8a296efe486196597d915792e  - Added as a part of [v0.5.0](https://github.com/uber-go/mock/releases/tag/v0.5.0). It'll be a part of the generated mocks if we'd to use v0.5.0 going forward.